doc/getting-started.rst

```diff
@@ -349,7 +349,7 @@ Buildout environment
 --------------------
 
 A Buildout environment includes the operating system and the Python
-installation it's run with. The more a buildout depends on its
+installation Buildout is executed with. The more a buildout depends on its
```

Comment:
"executed" is perhaps a bit more technically correct, but "run" seems fine enough to me.

Reply:
Okidoks!